### PR TITLE
Add reviews link to footer and footer spec

### PR DESCRIPTION
**Pull request by irwalker**
[vimaly](https://vimaly.com/#j8mqm/t/www_Add_Reviews_link_to_footer/Xc1vCD2gUb4AYNzCY)

image

### CODE REVIEW

**Review by CitationNerded — APPROVED** [review]
Nice one - looks just like my attempt did but i couldnt get the app running to verify

File: spec/features/footer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+describe 'Footer', type: :feature do
```
Comment (kylorhall) [inline spec/features/footer_spec.rb:3]: 👍

File: source/partials/_site_footer.html.erb [hunk 2]
```diff
@@ -80,6 +80,12 @@
           Pricing
         </a>
       </li>
+
+      <li role="menuitem">
+        <a href="<%= localize_url('reviews', locale_id: locale_obj[:id]) %>" class="footer__link" title="<%= locale_page(page: 'reviews')[:page_title] %>">
```
Comment (kylorhall) [inline source/partials/_site_footer.html.erb:85]: I don't believe we will have a `locale_page(page: 'reviews')[:page_title]`.  This likely needs its own title.

**Review by kylorhall — CHANGES_REQUESTED** [review]
Yes, but not quite (I think).  You can see what it is on staging though.

**Review by kylorhall — APPROVED** [review]
See what happens~!

**Comment by irwalker** [comment]
Getting this to work on mobile was weirdly hard. If you add or take anything away from the list, the layout no longer works on smaller screens.

Sam and I reformatted the list so it looks better on mobile, and OK on desktop. What are your thoughts pls @kylorhall ? Other option is to rebuild these elements using a table or something because the `li` way its done is a 'mare

Desktop
image

Tablet
image

Mobile
image

**Pull request merged**